config/src/macros.rs

```diff
@@ -94,3 +94,92 @@ macro_rules! impl_figment_convert {
         }
     };
 }
+/// A macro to implement converters from a type to [`Config`] and [`figment::Figment`]
```

Comment:
let's keep the docs simple here and just not that this is the same thing but if there's no `root: Option<PathBug>` field